cradle-admin-tool-http/README.md

```diff
@@ -46,6 +46,26 @@ spec:
     page-recheck-interval: 60 
 ```
 
+## Scripts
+
+### cradle-admin-tool-http/scripts/update-pages-comment.sh
```

Comment:
suggestion: I would add a list of non-standard utilities (e.g. jq) that are required for the script to work